modules/002-deckhouse/openapi/config-values.yaml

```diff
@@ -287,87 +293,111 @@ properties:
 
   registry:
     type: object
-    default:
-      mode: Unmanaged
+    x-doc-search: "registry, docker, container, images, credentials, private registry"
+    description: |
+      Settings for accessing the container registry with Deckhouse images.
 
+      This parameter allows you to configure how Deckhouse connects to the container registry to pull its own component images.
+      You can either rely on pre-configured access on cluster nodes (`Unmanaged` mode) or provide Deckhouse with direct access credentials (`Direct` mode).
     x-examples:
       - mode: Unmanaged
       - mode: Direct
         direct:
           imagesRepo: registry.deckhouse.io/deckhouse/ee
-          license: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-
+          license: "DECKHOUSE_LICENSE_KEY"
     properties:
       mode:
         type: string
         description: |
-          Registry mode
+          The mode for accessing the container registry with Deckhouse images.
+
+          - `Unmanaged` — Deckhouse does not manage access to the container registry. This mode assumes that the cluster nodes are already configured to access the required container registry (e.g., through CRI daemon settings or pre-pulled images). This is the default mode.
```

Comment:
That sentence is not true.

In `Unmanaged` mode Deckhouse is manages it's own access to registry, but it uses registry settings, which are passed during cluster bootstrap process.

Changing registry still possible by using `change-registry` helper

Also `Unamanaged` mode have downside on registry change - it is provokes full restart of all deckhouse components, which not true for `Direct` mode